submit_cgap/submission.py

```diff
@@ -108,39 +108,48 @@ def get_defaulted_institution(institution, user_record):
     """
     Returns the given institution or else if none is specified, it tries to infer an institution.
 
+    NOTE: As of the new permissions change, there is no way to infer an institution. One might be added later.
+          For now, there we just raise an error if no explicit institution was supplied.
+
     :param institution: the @id of an institution
     :param user_record: the user record for the authorized user
     :return: the @id of an institution to use
     """
-
+    ignored(user_record)
     if not institution:
-        submits_for = user_record.get('submits_for', [])
-        if len(submits_for) == 0:
-            raise SyntaxError("Your user profile declares no institution"
-                              " on behalf of which you are authorized to make submissions.")
-        elif len(submits_for) > 1:
-            raise SyntaxError("You must use --institution to specify which institution you are submitting for"
-                              " (probably one of: %s)." % ", ".join([x['@id'] for x in submits_for]))
-        else:
-            institution = submits_for[0]['@id']
-            show("Using institution:", institution)
+        # Ref: https://hms-dbmi.atlassian.net/browse/C4-371
+        raise SyntaxError("There not currently a way to infer an institution for user,"
+                          " so you must specify --institution explicitly.")
     return institution
 
 
 def get_defaulted_project(project, user_record):
     """
     Returns the given project or else if none is specified, it tries to infer a project.
 
-    :param project: the @id of a project
+    :param project: the @id of an project, or None
     :param user_record: the user record for the authorized user
-    :return: the @id of a project to use
+    :return: the @id of an project to use
```

Comment:
Why the change from `a` to `an`? Isn't `a` correct?

Reply:
It's far weirder than that. It's that the definitions of the tester for institution and project almost swapped places, so I borrowed the tester for institution with name substitution, and I forgot to change an article.

Reply:
Uploading a fix.